peer/peer.go

```diff
@@ -644,10 +648,16 @@ func (p *Peer) SelectedTip() *daghash.Hash {
 	return selectedTip
 }
 
-// IsSyncCandidate returns whether or not this peer is a sync candidate.
+// SetSelectedTip sets the selected tip of the peer.
+func (p *Peer) SetSelectedTip(selectedTip *daghash.Hash) {
```

Comment:
I think it's a good idea that anywhere we have a selectedTip as hash - call it selectedTipHash.

Reply:
One exception I can think of - MsgSelectedTip and MsgGetSelectedTip can be without the Hash ending.